examples/precision_example.cpp

```diff
@@ -0,0 +1,222 @@
+/** @file precision_example.cpp
+
+    @brief Example illustrating the effect of quantization and rounding errors in the absence of sufficient precision.
+
+    This file is part of the G+Smo library.
+
+    This Source Code Form is subject to the terms of the Mozilla Public
+    License, v. 2.0. If a copy of the MPL was not distributed with this
+    file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+    Author(s): N. Kohl
+*/
+
+#include <gismo.h>
+#include <gsAssembler/gsPoissonAssembler.h>
```

Comment:
gsPoissonAssembler.h is already included inside gismo.h, please remove it form here.

Reply:
Fixed in [1933fd1](https://github.com/gismo/gismo/pull/662/commits/1933fd1ef02e805950ad6d8166304e512b5ae796).